pkg/vtpm/swtpm-vtpm/src/main.go

```diff
@@ -24,21 +26,68 @@ import (
 )
 
 const (
-	swtpmPath            = "/usr/bin/swtpm"
-	stateEncryptionKey   = "/run/swtpm/%s.binkey"
-	swtpmLogPath         = "/run/swtpm/%s.log"
-	swtpmIsEncryptedPath = "/run/swtpm/%s.encrypted"
-	swtpmStatePath       = "/persist/swtpm/tpm-state-%s"
-	maxInstances         = 10
-	maxIDLen             = 128
+	swtpmPath    = "/usr/bin/swtpm"
+	purgeReq     = "purge"
+	deleteReq    = "delete"
+	launchReq    = "launch"
+	maxInstances = 10
+	maxIDLen     = 128
+	maxWaitTime  = 3 //seconds
 )
 
 var (
 	liveInstances int
 	log           *base.LogObject
+	pids          = make(map[string]int, 0)
+	// These are defined as vars to be able to mock them in tests
+	stateEncryptionKey   = "/run/swtpm/%s.binkey"
+	swtpmIsEncryptedPath = "/run/swtpm/%s.encrypted"
+	swtpmStatePath       = "/persist/swtpm/tpm-state-%s"
```

Comment:
Those are only used in vTPM service, there is no need for other parts of EVE to know about these locations at the moment.

Reply:
Some logic in EVE checks all the EVE-controlled files at the start. It would be useful to know that EVE has created something there.

And in general, it would be nice to have all the EVE-controlled files listed in one place. It will simplify debugging in some cases.